stl/inc/xutility

```diff
@@ -435,8 +435,8 @@ struct _Get_rebind_alias<_Ty, _Other, void_t<typename _Ty::template rebind<_Othe
 
 #if _HAS_CXX20
 _EXPORT_STD template <class _Ty, class... _Types>
-    requires requires(_Ty* _Location, _Types&&... _Args) {
-        ::new (static_cast<void*>(_Location)) _Ty(_STD forward<_Types>(_Args)...); // per LWG-3888
+    requires requires(void* _VoidPtr, _Types&&... _Args) {
+        ::new (_VoidPtr) _Ty(_STD forward<_Types>(_Args)...); // per LWG-3888

```

Comment:
The lengthy `static_cast<void*>(_Location)` was intentional. It was originally added in #3475 to speculatively implement LWG-3888. The simplified constraints conform to current standard specification, but have the speculative resolution of LWG-3888 dropped.

Reply:
It appears that the `P0784R7_library_support_for_more_constexpr_containers` failure is related to this.

Reply:
Thanks for adding the test to keep me from breaking this.

Reply:
It is still in the description

Reply:
> It is still in the description

Fixed.